source/documentation/3.9/rspec-expectations/RSpec/Matchers/BuiltIn/RespondTo.html

```diff
@@ -224,6 +225,29 @@ <h2>
   
     <span class="summary_desc"><div class='inline'></div></span>
   
+</li>
+
+      
+        <li class="public ">
+  <span class="summary_signature">
+    
+      <a href="#ignoring_method_signature_failure!-instance_method" title="#ignoring_method_signature_failure! (instance method)">#<strong>ignoring_method_signature_failure!</strong>  &#x21d2; Object </a>
+    
+
+    
+  </span>
+  
+  
+  
+  
+  
+  
+  <span class="private note title">private</span>
+
+  
+    <span class="summary_desc"><div class='inline'><p>Used by other matchers to suppress a check.</p>
+</div></span>
```

Comment:
Some of this shouldn't be here, possibly from the wrong version?

Reply:
It is because it is included in 3.9.1. 
https://github.com/rspec/rspec-expectations/blob/d671f8854532168b5e57c0479b5e6e5a61bc6cc8/lib/rspec/matchers/built_in/respond_to.rb#L107

The current documentation version is the 3.9.0
http://rspec.info/documentation/3.9/rspec-expectations/file.Changelog.html